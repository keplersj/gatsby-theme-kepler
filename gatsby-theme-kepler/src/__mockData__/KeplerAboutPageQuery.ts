import { AboutPageQuery } from "../pages/about";

export const KeplerAboutPageQuery: AboutPageQuery = {
  biography: {
    childMdx: {
      frontmatter: {
        title: "Biography"
      },
      body: ""
    }
  },
  experience: {
    edges: [
      {
        node: {
          id: "example-experience-listing",
          childMdx: {
            id: "example-experience-listing-mdx",
            body: "",
            frontmatter: {
              title: "Example Workplace",
              position: "Sample Position",
              /* eslint-disable @typescript-eslint/camelcase */
              rawEnd: "2018-12-18",
              end_date: "Dec 2018",
              rawStart: "2017-12-18",
              start_date: "Dec 2017"
              /* eslint-enable @typescript-eslint/camelcase */
            }
          }
        }
      }
    ]
  },
  education: {
    edges: [
      {
        node: {
          id: "example-education-listing",
          childMdx: {
            id: "example-education-listing-mdx",
            body: "",
            frontmatter: {
              title: "Example Institution",
              degree: "Sample Degree",
              /* eslint-disable @typescript-eslint/camelcase */
              rawEnd: "2018-12-18",
              end_date: "Dec 2018",
              rawStart: "2017-12-18",
              start_date: "Dec 2017"
              /* eslint-enable @typescript-eslint/camelcase */
            }
          }
        }
      }
    ]
  },
  skills: {
    childMdx: {
      frontmatter: {
        title: "Skills"
      },
      body: ""
    }
  }
};
