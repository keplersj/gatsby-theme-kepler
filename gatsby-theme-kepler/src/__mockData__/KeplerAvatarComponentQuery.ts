import { AvatarComponentQuery } from "../components/Avatar";

export const KeplerAvatarComponentQuery: AvatarComponentQuery = {
  image: {
    childImageSharp: {
      sqip: { dataURI: "data+image/svg+xml" },
      fixed: {
        width: 200,
        height: 200,
        src: "/static/example/1/avatar.jpg",
        srcSet: `/static/example/2/avatar.jpg 1x,
                /static/example/3/avatar.jpg 1.5x,
                /static/example/4/avatar.jpg 2x`,
      },
    },
  },
};
