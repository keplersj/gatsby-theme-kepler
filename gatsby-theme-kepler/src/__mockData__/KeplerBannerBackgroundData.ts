import { BannerData } from "../components/BannerBackground";

const sampleBase64 =
  "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAPABQDASIAAhEBAxEB/8QAGAAAAgMAAAAAAAAAAAAAAAAAAAYBAgX/xAAVAQEBAAAAAAAAAAAAAAAAAAAAAf/aAAwDAQACEAMQAAAB1ZUrU0CyL//EABoQAAICAwAAAAAAAAAAAAAAAAACARIREyH/2gAIAQEAAQUCiFMIUg6XY2Mf/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAwEBPwE//8QAFhEBAQEAAAAAAAAAAAAAAAAAAAER/9oACAECAQE/AW1//8QAGBAAAwEBAAAAAAAAAAAAAAAAAAIyEDH/2gAIAQEABj8C3pbFsf/EAB4QAAEDBAMAAAAAAAAAAAAAAAABEUEhUWFxkdHx/9oACAEBAAE/IcnggdtoLfKJjfce4f/aAAwDAQACAAMAAAAQGw//xAAWEQEBAQAAAAAAAAAAAAAAAAABABH/2gAIAQMBAT8QAsL/xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAgEBPxBWH//EABwQAQACAgMBAAAAAAAAAAAAAAEAESFRcbHRwf/aAAgBAQABPxBpi3Fuos5FpCIbpBzBcLAlYNX9gZ9/s//Z";

export const KeplerBannerBackgroundData: BannerData = {
  backdrop: {
    childImageSharp: {
      fluid: {
        base64: sampleBase64,
        aspectRatio: 1.333,
        src: "/static/banner_example/1/banner.jpg",
        srcSet: "/static/banner_example/1/banner.jpg 1x",
        srcWebp: "/static/banner_example/2/banner.jpg",
        srcSetWebp: "/static/banner_example/2/banner.jpg 1x",
        sizes: "(max-width: 4608px) 100vw, 4608px"
      }
    }
  },
  backdropDark: {
    childImageSharp: {
      fluid: {
        base64: sampleBase64,
        aspectRatio: 1.333,
        src: "/static/banner_example/1/banner_dark.jpg",
        srcSet: "/static/banner_example/1/banner_dark.jpg 1x",
        srcWebp: "/static/banner_example/2/banner_dark.webp",
        srcSetWebp: "/static/banner_example/2/banner_dark.webp 1x",
        sizes: "(max-width: 4608px) 100vw, 4608px"
      }
    }
  },
  desktop: {
    childImageSharp: {
      fluid: {
        base64: sampleBase64,
        aspectRatio: 1.333,
        src: "/static/banner_example/3/banner.jpg",
        srcSet: "/static/banner_example/3/banner.jpg 1x",
        srcWebp: "/static/banner_example/4/banner.webp",
        srcSetWebp: "/static/banner_example/4/banner.webp 1x",
        sizes: "(max-width: 4608px) 100vw, 4608px"
      }
    }
  },
  desktopDark: {
    childImageSharp: {
      fluid: {
        base64: sampleBase64,
        aspectRatio: 1.333,
        src: "/static/banner_dark_example/3/banner_dark.jpg",
        srcSet: "/static/banner_dark_example/3/banner_dark.jpg 1x",
        srcWebp: "/static/banner_dark_example/4/banner_dark.webp",
        srcSetWebp: "/static/banner_dark_example/4/banner_dark.webp 1x",
        sizes: "(max-width: 4608px) 100vw, 4608px"
      }
    }
  }
};
