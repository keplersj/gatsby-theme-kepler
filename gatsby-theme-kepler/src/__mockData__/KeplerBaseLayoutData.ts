import { BaseLayoutData } from "../components/BaseLayout";

export const KeplerBaseLayoutData: BaseLayoutData = {
  file: {
    childImageSharp: {
      fixed: {
        base64: "",
        width: 100,
        height: 100,
        src: "",
        srcSet: ""
      }
    }
  },
  site: {
    siteMetadata: {
      title: "Kepler Sticka-Jones",
      description: "",
      siteUrl: "https://site.dev",
      social: [
        {
          name: "Twitter",
          id: "@test",
          url: "https://twitter.com/test"
        },
        {
          name: "GitHub",
          id: "test",
          url: "https://github.com/test"
        }
      ]
    }
  }
};
