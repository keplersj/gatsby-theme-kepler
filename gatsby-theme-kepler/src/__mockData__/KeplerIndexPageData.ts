import { IndexPageData } from "../pages/index";

export const KeplerIndexPageData: IndexPageData = {
  site: {
    siteMetadata: {
      title: "Example Site",
      description: "This is an example of a personal web site.",
      siteUrl: "https://site.dev",
      social: [
        {
          name: "Email",
          id: "test@example.dev",
          url: "mailto:test@example.dev",
          isProfile: false
        },
        {
          name: "PGP",
          id: "XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX",
          url: "https://example.dev/pub.asc",
          isProfile: false
        },
        {
          name: "GitHub",
          id: "test",
          url: "https://github.dev/test",
          isProfile: true
        },
        {
          name: "Twitter",
          id: "@test",
          url: "https://twitter.dev/test",
          isProfile: true
        },
        {
          name: "LinkedIn",
          id: "test",
          url: "https://linkedin.dev/in/test",
          isProfile: true
        },
        {
          name: "Keybase",
          id: "test",
          url: "https://keybase.dev/test",
          isProfile: true
        },
        {
          name: "Instagram",
          id: "@test",
          url: "https://instagram.dev/test",
          isProfile: true
        },
        {
          name: "Unknown Platform of the Future",
          id: "test",
          url: "https://ü§∑üèº‚Äç‚ôÇÔ∏è.dev/test",
          isProfile: true
        }
      ],
      nav: [
        {
          name: "Blog",
          url: "/blog"
        },
        {
          name: "Portfolio",
          url: "/portfolio"
        },
        {
          name: "About",
          url: "/about"
        }
      ]
    }
  },
  biography: {
    childMdx: {
      frontmatter: {
        title: "Biography"
      },
      body:
        "I am a computer programmer and college student based out of Salt Lake City, with experience in entrepreneurship, student leadership, and open source software development."
    }
  },
  metadataImage: {
    childImageSharp: {
      fixed: {
        base64: "",
        width: 100,
        height: 100,
        src: "",
        srcSet: ""
      }
    }
  },
  blogPosts: {
    edges: [
      {
        node: {
          id: "post-1",
          title: "Test Post (without Image)",
          slug: "/blog/01-post-without-image",
          date: "Dec 18, 2018",
          rawDate: "2018-12-18",
          excerpt: "This is an example of a blog post without a featured image."
        }
      },
      {
        node: {
          id: "post-2",
          title: "Test Post (with Image)",
          slug: "/blog/02-post-with-image",
          date: "Dec 18, 2019",
          rawDate: "2018-12-18",
          excerpt: "This is an example of a blog post with a featured image.",
          featuredImage: {
            childImageSharp: {
              fluid: {
                aspectRatio: 1.0,
                src: "/static/image-1/image.jpeg",
                srcSet: "/static/image-1/image.jpeg 1x",
                sizes: "(min-width 100px)"
              }
            }
          }
        }
      }
    ]
  }
};
