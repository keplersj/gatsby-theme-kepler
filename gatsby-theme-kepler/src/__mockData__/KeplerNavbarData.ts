import { NavbarData } from "../components/Navbar";

export const KeplerNavbarData: NavbarData = {
  site: {
    siteMetadata: {
      title: "Kepler Sticka-Jones",
      nav: [
        {
          name: "Blog",
          url: "/blog"
        },
        {
          name: "Projects",
          url: "/projects"
        },
        {
          name: "About",
          url: "/about"
        }
      ]
    }
  }
};
