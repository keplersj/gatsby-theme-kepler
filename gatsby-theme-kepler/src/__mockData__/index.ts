export { KeplerAvatarComponentQuery } from "./KeplerAvatarComponentQuery";
export { KeplerBannerBackgroundData } from "./KeplerBannerBackgroundData";
export { KeplerBaseLayoutData } from "./KeplerBaseLayoutData";
export { KeplerNavbarData } from "./KeplerNavbarData";
