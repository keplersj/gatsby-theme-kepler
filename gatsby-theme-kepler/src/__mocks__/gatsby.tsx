import * as React from "react";
const gatsby = jest.requireActual("gatsby");

module.exports = {
  ...gatsby,
  graphql: jest.fn(),
  Link: jest.fn(
    ({
      // these props are invalid for an `a` tag
      /* eslint-disable @typescript-eslint/no-unused-vars */
      /* eslint-disable no-unused-vars */
      activeClassName,
      activeStyle,
      getProps,
      innerRef,
      partiallyActive,
      ref,
      replace,
      /* eslint-enable @typescript-eslint/no-unused-vars */
      /* eslint-enable no-unused-vars */
      to,
      ...rest
      // eslint-disable-next-line jsx-a11y/anchor-has-content
    }): React.ReactElement => <a {...rest} href={to} />
  ),
  StaticQuery: jest.fn(),
  useStaticQuery: jest.fn()
};
