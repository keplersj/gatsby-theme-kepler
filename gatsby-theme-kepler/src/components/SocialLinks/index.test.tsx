import * as React from "react";
import renderer from "react-test-renderer";
import { useStaticQuery } from "gatsby";
import { SocialLinks } from ".";
import { KeplerSocialLinksQuery } from "../../__mockData__";

beforeEach((): void => {
  (useStaticQuery as jest.Mock).mockImplementation(
    (): object => KeplerSocialLinksQuery
  );
});

describe("SocialLinks", (): void => {
  it("renders correctly", (): void => {
    const tree = renderer.create(<SocialLinks />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
