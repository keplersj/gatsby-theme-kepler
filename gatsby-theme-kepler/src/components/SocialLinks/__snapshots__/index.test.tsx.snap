// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialLinks renders correctly 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.emotion-1 {
  padding: 0.5em;
}

<address
  className="emotion-0"
  role="list"
>
  <a
    className="emotion-1"
    href="mailto:test@example.dev"
    id="test@example.dev"
    name="Email"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 512 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Email: test@example.dev
      </title>
      <path
        d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://example.dev/pub.asc"
    id="XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX"
    name="PGP"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 512 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        PGP: XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX
      </title>
      <path
        d="M512 176.001C512 273.203 433.202 352 336 352c-11.22 0-22.19-1.062-32.827-3.069l-24.012 27.014A23.999 23.999 0 0 1 261.223 384H224v40c0 13.255-10.745 24-24 24h-40v40c0 13.255-10.745 24-24 24H24c-13.255 0-24-10.745-24-24v-78.059c0-6.365 2.529-12.47 7.029-16.971l161.802-161.802C163.108 213.814 160 195.271 160 176 160 78.798 238.797.001 335.999 0 433.488-.001 512 78.511 512 176.001zM336 128c0 26.51 21.49 48 48 48s48-21.49 48-48-21.49-48-48-48-48 21.49-48 48z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://github.dev/test"
    id="test"
    name="GitHub"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 496 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        GitHub: test
      </title>
      <path
        d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://twitter.dev/test"
    id="@test"
    name="Twitter"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 512 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Twitter: @test
      </title>
      <path
        d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://linkedin.dev/in/test"
    id="test"
    name="LinkedIn"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 448 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        LinkedIn: test
      </title>
      <path
        d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://keybase.dev/test"
    id="test"
    name="Keybase"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 448 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Keybase: test
      </title>
      <path
        d="M286.17 419a18 18 0 1 0 18 18 18 18 0 0 0-18-18zm111.92-147.6c-9.5-14.62-39.37-52.45-87.26-73.71q-9.1-4.06-18.38-7.27a78.43 78.43 0 0 0-47.88-104.13c-12.41-4.1-23.33-6-32.41-5.77-.6-2-1.89-11 9.4-35L198.66 32l-5.48 7.56c-8.69 12.06-16.92 23.55-24.34 34.89a51 51 0 0 0-8.29-1.25c-41.53-2.45-39-2.33-41.06-2.33-50.61 0-50.75 52.12-50.75 45.88l-2.36 36.68c-1.61 27 19.75 50.21 47.63 51.85l8.93.54a214 214 0 0 0-46.29 35.54C14 304.66 14 374 14 429.77v33.64l23.32-29.8a148.6 148.6 0 0 0 14.56 37.56c5.78 10.13 14.87 9.45 19.64 7.33 4.21-1.87 10-6.92 3.75-20.11a178.29 178.29 0 0 1-15.76-53.13l46.82-59.83-24.66 74.11c58.23-42.4 157.38-61.76 236.25-38.59 34.2 10.05 67.45.69 84.74-23.84.72-1 1.2-2.16 1.85-3.22a156.09 156.09 0 0 1 2.8 28.43c0 23.3-3.69 52.93-14.88 81.64-2.52 6.46 1.76 14.5 8.6 15.74 7.42 1.57 15.33-3.1 18.37-11.15C429 443 434 414 434 382.32c0-38.58-13-77.46-35.91-110.92zM142.37 128.58l-15.7-.93-1.39 21.79 13.13.78a93 93 0 0 0 .32 19.57l-22.38-1.34a12.28 12.28 0 0 1-11.76-12.79L107 119c1-12.17 13.87-11.27 13.26-11.32l29.11 1.73a144.35 144.35 0 0 0-7 19.17zm148.42 172.18a10.51 10.51 0 0 1-14.35-1.39l-9.68-11.49-34.42 27a8.09 8.09 0 0 1-11.13-1.08l-15.78-18.64a7.38 7.38 0 0 1 1.34-10.34l34.57-27.18-14.14-16.74-17.09 13.45a7.75 7.75 0 0 1-10.59-1s-3.72-4.42-3.8-4.53a7.38 7.38 0 0 1 1.37-10.34L214 225.19s-18.51-22-18.6-22.14a9.56 9.56 0 0 1 1.74-13.42 10.38 10.38 0 0 1 14.3 1.37l81.09 96.32a9.58 9.58 0 0 1-1.74 13.44zM187.44 419a18 18 0 1 0 18 18 18 18 0 0 0-18-18z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://instagram.dev/test"
    id="@test"
    name="Instagram"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 448 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Instagram: @test
      </title>
      <path
        d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
      />
    </svg>
  </a>
  <a
    className="emotion-1"
    href="https://🤷🏼‍♂️.dev/test"
    id="test"
    name="Unknown Platform of the Future"
    rel="noopener noreferrer"
    role="listitem"
    target="_blank"
  >
    <svg
      fill="currentColor"
      height="1.5em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 448 512"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Unknown Platform of the Future: test
      </title>
      <path
        d="M448 80v352c0 26.51-21.49 48-48 48H48c-26.51 0-48-21.49-48-48V80c0-26.51 21.49-48 48-48h352c26.51 0 48 21.49 48 48zm-88 16H248.029c-21.313 0-32.08 25.861-16.971 40.971l31.984 31.987L67.515 364.485c-4.686 4.686-4.686 12.284 0 16.971l31.029 31.029c4.687 4.686 12.285 4.686 16.971 0l195.526-195.526 31.988 31.991C358.058 263.977 384 253.425 384 231.979V120c0-13.255-10.745-24-24-24z"
      />
    </svg>
  </a>
</address>
`;
